lefeef/chain.json

```diff
@@ -0,0 +1,134 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "Lefeef",
+    "chain_type": "cosmos",
+    "chain_id": "lefeef-mainnet",
+    "website": "https://lefeef.net/",
+    "pretty_name": "lefeef",
+    "status": "live",
+    "network_type": "mainnet",
+    "bech32_prefix": "lefeef",
```

Comment:
Is it recommended to register this prefix here
https://github.com/satoshilabs/slips/blob/master/slip-0173.md